_maps/shuttles/emergency_fish.dmm

```diff
@@ -0,0 +1,1447 @@
+"lW" = (
+/turf/open/chasm,
+/area/shuttle/escape)
+"mD" = (
+/obj/item/toy/beach_ball/branded,
+/turf/open/water/beach/biodome,
+/area/shuttle/escape)
+"nz" = (
+/obj/machinery/door/airlock/medical/glass,
+/turf/open/floor/mineral/titanium/blue,
+/area/shuttle/escape)
+"nB" = (
+/obj/machinery/vending/wallmed/directional/north{
+	use_power = 0
+	},
+/obj/machinery/light/directional/north,
+/obj/effect/turf_decal/tile/dark/half/contrasted{
+	dir = 1
+	},
+/turf/open/floor/wood/tile,
+/area/shuttle/escape)
+"nN" = (
+/obj/effect/spawner/structure/window/reinforced/shuttle,
+/turf/open/floor/wood/tile,
+/area/shuttle/escape)
+"nY" = (
+/obj/structure/table/wood,
+/obj/item/book/fish_catalog{
+	pixel_y = 2
+	},
+/turf/open/floor/wood/tile,
+/area/shuttle/escape)
+"oc" = (
+/obj/effect/turf_decal/tile/blue{
+	dir = 8
+	},
+/obj/effect/turf_decal/tile/blue{
+	dir = 4
+	},
+/obj/structure/table/glass,
+/obj/item/storage/medkit/regular{
+	pixel_x = 2;
+	pixel_y = 3
+	},
+/obj/machinery/light/directional/south,
+/turf/open/floor/iron/white,
+/area/shuttle/escape)
+"og" = (
+/obj/effect/turf_decal/tile/dark/half/contrasted{
+	dir = 8
+	},
+/obj/item/kirbyplants/random,
+/turf/open/floor/wood/tile,
+/area/shuttle/escape)
+"oA" = (
+/obj/structure/sign/nanotrasen,
+/turf/closed/wall/mineral/titanium/nodiagonal,
+/area/shuttle/escape)
+"oI" = (
+/obj/structure/chair/comfy/shuttle{
+	dir = 8
+	},
+/obj/machinery/light/directional/north,
+/turf/open/floor/mineral/titanium/tiled/white,
+/area/shuttle/escape)
+"pa" = (
+/obj/effect/turf_decal/tile/dark_red/opposingcorners,
+/obj/machinery/vending/wallmed/directional/south,
+/obj/structure/railing{
+	dir = 1
+	},
+/obj/effect/turf_decal/stripes{
+	dir = 1
+	},
+/turf/open/floor/mineral/plastitanium,
+/area/shuttle/escape)
+"pk" = (
+/obj/effect/turf_decal/tile/blue{
+	dir = 1
+	},
+/obj/effect/turf_decal/tile/blue{
+	dir = 4
+	},
+/obj/effect/turf_decal/tile/blue{
+	dir = 8
+	},
+/obj/machinery/light/directional/north,
+/turf/open/floor/iron/white,
+/area/shuttle/escape)
+"pQ" = (
+/obj/effect/turf_decal/tile/dark/half/contrasted{
+	dir = 4
+	},
+/obj/structure/extinguisher_cabinet/directional/east,
+/turf/open/floor/wood/tile,
+/area/shuttle/escape)
+"qe" = (
+/obj/structure/aquarium{
+	fluid_type = "Saltwater"
+	},
+/obj/effect/turf_decal/tile/dark/anticorner/contrasted{
+	dir = 4
+	},
+/turf/open/floor/wood/tile,
+/area/shuttle/escape)
+"qA" = (
+/turf/open/floor/mineral/titanium/tiled/white,
+/area/shuttle/escape)
+"qO" = (
+/obj/effect/turf_decal/tile/dark/half/contrasted{
+	dir = 4
+	},
+/obj/structure/table/wood,
+/obj/item/fishing_line{
+	layer = 4;
+	pixel_x = 6;
+	pixel_y = 3
+	},
+/obj/item/fishing_hook{
+	pixel_y = 3;
+	layer = 5
+	},
+/obj/item/bait_can/worm{
+	pixel_x = -5;
+	pixel_y = 13
+	},
+/turf/open/floor/wood/tile,
+/area/shuttle/escape)
+"rj" = (
+/obj/docking_port/mobile/emergency{
+	name = "Angler's Choice emergency shuttle";
+	height = 13;
+	width = 27;
+	dwidth = 6
+	},
```

Comment:
```suggestion
	name = "Angler's Choice emergency shuttle"
	},
```

Reply:
When I wrote this (a few months ago) I had to make this edit or the brig disappeared
I'm glad that sems to have been fixed in the intervening time